python/paddle/v2/framework/tests/test_softmax_op.py

```diff
@@ -23,6 +23,32 @@ def setUp(self):
         self.Y = np.apply_along_axis(stable_softmax, 1, self.X)
 
 
+class TestSoftmaxGradOp1(unittest.TestCase):
```

Comment:
What does the class name have suffix `1`?  Should we give it a more specific name?